portfolio/src/main/webapp/intro.html

```diff
@@ -0,0 +1,123 @@
+            </div>
+
+            
+        </div>
+
+        <div class="twoColumnRight">
+            <h2> Hi, I am Yi Lyu :D</h2>
+
+            <!--Education-->
+            <div class="block container">
+                <div class="leftColumn"> Education <img src="images/education.png" width="20%"> </div>
+                <div class="middleColumn">
+                    Bachelor, major in Information Engineering <br>
+                    Zhiyuan Honor Track
+                </div>
+                <div class="rightColumn"> 
+                    2017.9-2021.6
+                </div>
+            </div>
+
+            <!-- Experience-->
+            <div class="block container">
+                <div class="leftColumn"> Experience <img src="images/suitcase.png" width="15%"></div>
+                <div class="middleColumn"> SWE intern in Azure Storage, Microsoft<br>STEP intern in Data Platform, Google
+                </div>
+                <div class="rightColumn"> 2020.7-2020.10 <br> 2019.7-2019.10 </div>
+            </div>
+
+            <!--Project-->
+            <div class="block container">
+                <div class="leftColumn"> Project <img src="images/projects.png" width="20%"></div>
+                <div class="middleColumn">
+                    JOS, an OS kernel from Scratch <br>
+                    Go Intepreter <br>
+                    LSM Tree Database
+                </div>
+                <div class="rightColumn">
+                    <a href="https://github.com/isabella0428/JOS.git"> Github link </a> <br>
+                    <a href="https://github.com/isabella0428/Monkey.git"> Github link </a> <br>
+                    <a href="https://github.com/isabella0428/SJTU_SE121/tree/master/project1-kvstore"> Github link </a> <br>
+                </div>
+            </div>
+
+            <!--Interest-->
+            <div class="block container">
+                <div class="leftColumn"> Interests<img src="images/circle.png" width="15%"> </div>
+                <div class="middleColumn">
+                    American TV series <br>
+                    <br>
+                    <br>
+                    <br>
+                    Singing, Karaoke
+                </div>
+                <div class="rightColumn">
+                    Modern Family <br> 
+                    Big Bang Theory <br>
+                    Big Little Lies <br>
+                    <br>
+                    Go to Karaoke with me :)
+                </div>
+            </div>
+            
+            <div class="block container">
+                <div class="leftColumn">
+                    Programming Language <br>
+                    <br>
+                    Framework
+                </div>
+
+                <div class="middleColumn">
+                    C/C++, Python, Java, JavaScript, Python, HTML, CSS, SQL <br>
+                    <br>
+                    <br>
+                    Tensorflow,Keras, MATLAB, LATEX, FLASK,d3.js
```

Comment:
Nit: TensorFlow, LaTex, Flask, D3.js.

You may need to pay attention to those tech terms, though it doesn't affect anything, but it gives a sense about how careful you are :P.